website/docs/r/vpc_peering_options.html.markdown

```diff
@@ -0,0 +1,166 @@
+---
+layout: "aws"
+page_title: "AWS: aws_vpc_peering_connection_options"
+sidebar_current: "docs-aws-resource-vpc-peering-options"
+description: |-
+  Provides a resource to manage VPC peering connection options.
+---
+
+# aws_vpc_peering_connection_options
+
+Provides a resource to manage VPC peering connection options.
+
+~> **NOTE on VPC Peering Connections and VPC Peering Connection Options:** Terraform provides
+both a standalone VPC Peering Connection Options and a [VPC Peering Connection](vpc_peering.html)
+resource with `accepter` and `requester` attributes. Do not manage options for the same VPC peering
+connection in both a VPC Peering Connection resource and a VPC Peering Connection Options resource.
+Doing so will cause a conflict of options and will overwrite the options.
+
+Basic usage:
+
+```hcl
+resource "aws_vpc" "foo" {
+  cidr_block = "10.0.0.0/16"
+}
+
+resource "aws_vpc" "bar" {
+  cidr_block = "10.1.0.0/16"
+}
+
+resource "aws_vpc_peering_connection" "foo" {
+  vpc_id        = "${aws_vpc.foo.id}"
+  peer_vpc_id   = "${aws_vpc.bar.id}"
+  auto_accept   = true
+}
+
+resource "aws_vpc_peering_connection_options" "foo" {
+  vpc_peering_connection_id = "${aws_vpc_peering_connection.foo.id}"
+
+  accepter {
+    allow_remote_vpc_dns_resolution = true
+  }
+
+  requester {
+    allow_vpc_to_remote_classic_link = true
+    allow_classic_link_to_remote_vpc = true
+  }
+}
+```
+
+Basic cross-account usage:
+
+```hcl
+provider "aws" {
+  # Requester's credentials.
```

Comment:
It might be helpful to very explicitly alias these as `alias = "requester"` and `alias = "accepter"` so the resources are very explicit as well.

Reply:
Agreed. Will do.

Reply:
Verified that the new text works as expected:

```
$ terraform apply
data.aws_caller_identity.peer: Refreshing state...
aws_vpc.peer: Creating...
  assign_generated_ipv6_cidr_block: "" => "false"
  cidr_block:                       "" => "10.1.0.0/16"
  default_network_acl_id:           "" => "<computed>"
  default_route_table_id:           "" => "<computed>"
  default_security_group_id:        "" => "<computed>"
  dhcp_options_id:                  "" => "<computed>"
  enable_classiclink:               "" => "<computed>"
  enable_classiclink_dns_support:   "" => "<computed>"
  enable_dns_hostnames:             "" => "true"
  enable_dns_support:               "" => "true"
  instance_tenancy:                 "" => "<computed>"
  ipv6_association_id:              "" => "<computed>"
  ipv6_cidr_block:                  "" => "<computed>"
  main_route_table_id:              "" => "<computed>"
aws_vpc.main: Creating...
  assign_generated_ipv6_cidr_block: "" => "false"
  cidr_block:                       "" => "10.0.0.0/16"
  default_network_acl_id:           "" => "<computed>"
  default_route_table_id:           "" => "<computed>"
  default_security_group_id:        "" => "<computed>"
  dhcp_options_id:                  "" => "<computed>"
  enable_classiclink:               "" => "<computed>"
  enable_classiclink_dns_support:   "" => "<computed>"
  enable_dns_hostnames:             "" => "true"
  enable_dns_support:               "" => "true"
  instance_tenancy:                 "" => "<computed>"
  ipv6_association_id:              "" => "<computed>"
  ipv6_cidr_block:                  "" => "<computed>"
  main_route_table_id:              "" => "<computed>"
aws_vpc.main: Creation complete after 5s (ID: vpc-48684831)
aws_vpc.peer: Creation complete after 6s (ID: vpc-876343fe)
aws_vpc_peering_connection.peer: Creating...
  accept_status: "" => "<computed>"
  accepter.#:    "" => "<computed>"
  auto_accept:   "" => "false"
  peer_owner_id: "" => "000000000000"
  peer_region:   "" => "<computed>"
  peer_vpc_id:   "" => "vpc-876343fe"
  requester.#:   "" => "<computed>"
  tags.%:        "" => "1"
  tags.Side:     "" => "Requester"
  vpc_id:        "" => "vpc-48684831"
aws_vpc_peering_connection.peer: Creation complete after 2s (ID: pcx-47b7a92e)
aws_vpc_peering_connection_accepter.peer: Creating...
  accept_status:             "" => "<computed>"
  accepter.#:                "" => "<computed>"
  auto_accept:               "" => "true"
  peer_owner_id:             "" => "<computed>"
  peer_region:               "" => "<computed>"
  peer_vpc_id:               "" => "<computed>"
  requester.#:               "" => "<computed>"
  tags.%:                    "" => "1"
  tags.Side:                 "" => "Accepter"
  vpc_id:                    "" => "<computed>"
  vpc_peering_connection_id: "" => "pcx-47b7a92e"
aws_vpc_peering_connection_accepter.peer: Creation complete after 3s (ID: pcx-47b7a92e)
aws_vpc_peering_connection_options.accepter: Creating...
  accepter.#:                                           "" => "1"
  accepter.1102046665.allow_classic_link_to_remote_vpc: "" => "false"
  accepter.1102046665.allow_remote_vpc_dns_resolution:  "" => "true"
  accepter.1102046665.allow_vpc_to_remote_classic_link: "" => "false"
  requester.#:                                          "" => "<computed>"
  vpc_peering_connection_id:                            "" => "pcx-47b7a92e"
aws_vpc_peering_connection_options.requester: Creating...
  accepter.#:                                            "" => "<computed>"
  requester.#:                                           "" => "1"
  requester.1102046665.allow_classic_link_to_remote_vpc: "" => "false"
  requester.1102046665.allow_remote_vpc_dns_resolution:  "" => "true"
  requester.1102046665.allow_vpc_to_remote_classic_link: "" => "false"
  vpc_peering_connection_id:                             "" => "pcx-47b7a92e"
aws_vpc_peering_connection_options.requester: Creation complete after 1s (ID: pcx-47b7a92e)
aws_vpc_peering_connection_options.accepter: Creation complete after 1s (ID: pcx-47b7a92e)

Apply complete! Resources: 6 added, 0 changed, 0 destroyed.
```